models/standardized_models/stripe__standardized_models.yml

```
@@ -33,7 +33,7 @@ models:
       - name: product_type
         description: Product type
       - name: transaction_type
-        description: Balance transaction type
+        description: Balance transaction type. If refunds and charges are tied to one invoice, it's designated as 'refund+charge'.
```

```suggestion
        description: Balance transaction type. If refunds and charges are tied to one invoice, it's designated as 'charge + refund'.
```
Should we include the other  type(s) as well?

---

Good call.